<launch>

    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="usb_cam" />
        <param name="io_method" value="mmap"/>

        <param name="camera_info_url" type="string" value="file://$(find tcst_pkg)/yaml/cam_300_industry_20200518.yaml" />
    </node>
     <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
            <remap from="image" to="/usb_cam/image_raw"/>
            <param name="autosize" value="true" />
     </node>
    <node pkg="tcst_pkg" type="ioboard_node.py" name="ioboard_node" output="screen" >
    </node>
    <param name="open_go_to_object" value="0" />
    <param name="open_go_to_desire" value="0" />
    <param name="open_ibvs_flag" value="0" />
    <param name="open_go_desire_flag" value="0" />
    <param name="choose_next_point" value="0" />
    <param name="threshold_min" value="120" />
    <param name="color_min" value="120" />
    <node pkg="tcst_pkg" type="detect_box_and_pick_sucking.py" name="detect_box_and_pick_sucking" output="screen" >
    </node>
    
  <!-- </group> -->
  
</launch>